src/components/v5/common/TeamCard/TeamCard.tsx

```diff
@@ -70,7 +75,12 @@ const TeamCard: FC<TeamCardProps> = ({
               />
             </div>
           ) : (
-            <>
+            <Link
+              to={{
+                pathname: `/${colonyName}/${COLONY_CONTRIBUTORS_ROUTE}`,
+                search: `${links?.[0]?.to.search}`,
```

Comment:
I would suggest refactoring this line, as the `links` might update and is not a reliable source of truth for the parameter we want to use

So my suggestion would be to add an extra property to the `TeamCardProps` either `nativeId` or `searchParams`
and then update the `useTeams` hook to include either the `nativeId` or 
```
searchParams: {
  team: `${TEAM_SEARCH_PARAM}=${nativeId}`
}
```

and then use it here either as
`search: `?${TEAM_SEARCH_PARAM}=${nativeId}`,`
or
`search: `?${searchParams.team}`,`

Reply:
@mmioana done 🙂